### PR TITLE
feat: Sending emails with node.js

### PR DESCRIPTION
**Pull request by 41hulk**
Added a nodemailer and sent an email to @ceelogre on his email

### CODE REVIEW

**Review by ceelogre — CHANGES_REQUESTED** [review]
Good work Guy. This is a very useful feature you've added. Just one thing before I merge, can you add instructions on how to test what you've added in the Pull Request description? Thanks.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# NODEMAILER
+
+Foobar is a Python library for dealing with word pluralization.
+
```
Comment (ceelogre) [inline README.md:4]: Is this relevant?

Comment (41hulk) [reply README.md:4]: Sorry, it's a markup i used to create readme files and i didn't correct that part

File: package.json [hunk 2]
```diff
@@ -14,6 +14,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "nodemailer": "^6.4.13",
+    "nodemon": "^2.0.4"
```
Comment (ceelogre) [inline package.json:19]: `nodemon` is a development dependency. It should be moved to `devDependencies`

**Review by ceelogre — CHANGES_REQUESTED** [review]

**Review by ceelogre — APPROVED** [review]

**Pull request merged**